components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -163,4 +163,19 @@ -(void)testTitleVisibility {
   XCTAssert(self.bottomNavBar.itemViews.lastObject.label.isHidden);
 }
 
+-(void)testViewForItem {
+  UITabBarItem *item1 = [[UITabBarItem alloc] initWithTitle:@"1" image:nil tag:0];
```

Comment:
Please mark each section of your test with comments for the "Given" (set-up), "When" (condition being tested), and "Then" (assertions) portions of your test.  See `testSelectedItemAfterReset` for an example in this file.

Reply:
Sounds good.